Robus/src/reception.c

```diff
@@ -168,6 +169,7 @@ void Recep_GetData(volatile uint8_t *data)
             {
                 MsgAlloc_Reset();
                 ctx.tx.status = TX_DISABLE;
+                Robus_SetNodeDetected(NETWORK_LINK_CONNECT);
```

Comment:
Please use the `ctx` global var avoiding function call on IRQ.

Reply:
I think using the ctx struct to save this variable is not the best solution as it creates a strong coupling between ctx and network state which have distinct purposes. If you care about the time spend in this function, the function can be declared as inline. But I think the memset call MsgAlloc_Reset() is more time consuming than this function.

Anyway, I propose to declare this function as inlined.

Reply:
Things on Robus are network related, and everything is already on the ctx var. When we will separate them it will be better to don't have any exceptions.
We are not on Luos V3 yet 😉.

I'm still convinced that we should put it on ctx.
If you don't, I'm ok to inline it.

Reply:
Ok I moved network variables in ctx structure.